content/blogs/release-0-22.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Kestra 0.22 introduces Plugin Versioning, External Secrets, and Namespace Sharing
+description: Kestra 0.22 brings powerful new features including Plugin Versioning, External Secrets, and enhanced namespace sharing capabilities. This release focuses on enterprise-grade management features while improving developer experience with new plugins and Pebble functions.
```

Comment:
sooo much repetition 😭 

![image](https://github.com/user-attachments/assets/c9291f96-88ed-4ded-9817-126e20647833)
 can you revise @Ben8t? maybe list features only in description but cut from the intro i.e. in intro only point to the table?

Reply:
Oh didn't check the render; thought the meta description is not showing. definitely moving away the second paragraph